src/demo/spawn.ts

```diff
@@ -0,0 +1,283 @@
+import {Identity, Application} from 'openfin/_v2/main';
+import {ChannelClient} from 'openfin/_v2/api/interappbus/channel/client';
+import {_Window} from 'openfin/_v2/api/window/window';
+import {ApplicationOption} from 'openfin/_v2/api/application/applicationOption';
+import {WindowOption} from 'openfin/_v2/api/window/windowOption';
+
+export type Dictionary<T = string> = {
+    [key: string]: T
+};
+
+export interface Point<T = number> {
```

Comment:
Get it's just 4 lines, but I'd still be tempted to move this to it's own file, just to make it a little bit easier whenever we unify spawn.ts with the layouts equivalent and pull it into a shared module

Reply:
Time to bring in @pjbroadbent. Asked him about this yesterday since the layouts one references the `PointUtils` module from snap&dock. We agreed to just have it in here for now as there is really no other use for a Point in notifs and it didn't seem worth rewriting a load of the spawn stuff to not need it.

Reply:
IMO, until we have other uses for this, it makes sense to leave it here.

If we find that this type would be useful when doing toast positioning or something like that, then we can find a more sensible place for it. But until then, leave it in the one file that uses it?..

Reply:
@bryangaleOF, happy for me to mark resolved based on Phil's response?

Reply:
Sure thing